client/src/components/LoginPage.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import api from '../helpers/api';
+
+class LoginPage extends React.Component {
+    state ={
+        username: '',
+        password: ''
+    }
+    handleSubmit = async (e) => {
+        e.preventDefault();
+        console.log(this.state);
+        try {
+            const {username, password} = this.state;
+            const result = await api.post('/auth/login', {
+                username,
+                password,
+            })
+            this.setState({
+                username: '',
+                password: ''
+            })
+
+            // save token in localStorage
+            localStorage.setItem('token', result.data.token);
```

Comment:
great job setting your local storage. I'm so curious to see what this api helper function you made looks like 😮